.travis.yml

```diff
@@ -1,2 +1,8 @@
 language: c
+before_install:
+- wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && sudo dpkg -i erlang-solutions_1.0_all.deb
+- sudo apt-get update
+- sudo apt-get install elixir
+- sudo apt-get install erlang
```

Comment:
Can you squash lines 5-6 together? `sudo apt-get install elixir erlang`. Might be slightly faster, since apt only has to resolve the dependency graph once instead of twice.